metpetdb_api/install_metpetdb.sh

```diff
@@ -0,0 +1,323 @@
+#!/bin/bash
```

Comment:
Oh sorry about that, I accidentally committed the file. I didn't intend on doing that.